pkg/machine/hyperv/machine.go

```diff
@@ -205,7 +216,61 @@ RequiredBy=default.target
 		Name:     "ready.service",
 		Contents: machine.StrToPtr(fmt.Sprintf(ready, m.ReadyHVSock.Port)),
 	}
-	ign.Cfg.Systemd.Units = append(ign.Cfg.Systemd.Units, readyUnit)
+
+	// userNetwork is a systemd unit file that calls the vm helpoer utility
+	// needed to take traffic from a network vsock0 device to the actual vsock
+	// and onto the host
```

Comment:
Comment for later: I think eventually we'll need a folder full of systemd unit files for code readibility